src/node_enum.rs

```diff
@@ -692,15 +697,12 @@ impl NodeEnum {
                 }
                 NodeMut::GrantStmt(s) => {
                     let s = s.as_mut().unwrap();
-                    match protobuf::ObjectType::from_i32(s.objtype) {
-                        Some(protobuf::ObjectType::ObjectTable) => {
-                            s.objects.iter_mut().for_each(|n| {
-                                if let Some(n) = n.node.as_mut() {
-                                    iter.push((n.to_mut(), depth, Context::DDL));
-                                }
-                            });
-                        }
-                        _ => (),
+                    if let Some(protobuf::ObjectType::ObjectTable) = protobuf::ObjectType::from_i32(s.objtype) {
```

Comment:
I was just following the clippy suggestion, but it might be more clear as:
```rs
if protobuf::ObjectType::from_i32(s.objtype) == Some(protobuf::ObjectType::ObjectTable) {

Reply:
This LGTM, the slight benefit of using `if let` over a comparison with `==` is that this version doesn't require deriving `PartialEq` on anything, so it's a little more lean